src/views/r-page.html

```diff
@@ -10,11 +10,11 @@
 		<head>
 			<meta charset="utf-8">
 			<meta name="viewport" content="width=device-width, initial-scale=1">
-			<meta name="description" content="{{ description || 'A free, fast, and reliable Open Source CDN for npm and GitHub with the largest network and best performance among all CDNs. Serving more than 40 billion requests per month.'}}">
-			<meta name="keywords" content="jsdelivr, cdn, npm, github, js, javascript, free, open source, oss">
+			<meta name="description" content="{{ description || 'A free, fast, and reliable CDN for open source projects, deno, npm and GitHub with the largest network and best performance among all CDNs. Serving more than 80 billion requests per month.'}}">
```

Comment:
"for open source projects, deno, npm" etc doesn't make much sense, if you don't want to call it open source cdn, we can just drop that part and let it be "reliable CDN for deno, npm, ..."

Reply:
But we are not limited only for them. We support any open source project with our custom configs.

Reply:
Maybe but that's not the main focus, makes the text too complicated, and in no way explains what it means.

Reply:
how so? i think "a free cdn for open source projects" is pretty clear

Reply:
Yes but when you mix it with the other terms it gets long and "open source projects" doesn't really fit in there "CDN for open source projects, deno, npm, and GitHub"

Reply:
Maybe its not ideal but I dont see any other way. Our site is not very clear or Google friendly. 
This way the user will just look at "CDN for" and then the keyword relevant to him and understand that it fits his need.

Reply:
We have open source all over the place, including the main headline, it doesn't need to be everywhere.

Reply:
which always refers to us, not projects we host. Also this description is for Google where other mentions of oss are not visible